README.md

```diff
@@ -36,6 +36,7 @@ import { Contrast } from "@smockle/contrast";
 
 // Hexadecimal colors
 new Contrast("#000000", "#FFFFFF");
+  // -> { foreground: HexColor, background: HexColor, value: 21 }
```

Comment:
```suggestion
  // => {
  //      foreground: {
  //        value: "000000",
  //        R: { value: "00" },
  //        G: { value: "00" },
  //        B: { value: "00" },
  //      },
  //      background: {
  //        value: "FFFFFF",
  //        R: { value: "FF" },
  //        G: { value: "FF" },
  //        B: { value: "FF" },
  //      },
  //      value: 21,
  //    }
```

Reply:
Adding a suggestion per the idea you shared in https://github.com/smockle/contrast/pull/213#issuecomment-632696945 👍